protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -344,6 +344,14 @@ service TradingDataService {
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Markets"};
   }
 
+  // List successor markets
+  //
+  // Given a market id, return the full lineage of markets since inception, or all successor markets since and including
+  // the given market id.
```

Comment:
```suggestion
  // Given a market ID, return the full lineage of markets since inception, or all successor markets since and including
  // the given market ID.
```